src/cdk/a11y/a11y.md

```diff
@@ -0,0 +1,90 @@
+The `a11y` package provides a number of tools to improve accessibility, described below.
+
+### ListKeyManager
+`ListKeyManager` manages the active option in a list of items based on keyboard interaction.
+Intended to be used with components that correspond to a `role="menu"` or `role="listbox"` pattern.
+
+#### Basic usage
+Any component that uses a `ListKeyManager` will generally do three things:
+* Create a `@ViewChildren` query for the options being managed.
+* Initialize the `ListKeyManager`, passing in the options.
+* Forward keyboard events from the managed component to the `ListKeyManager`
+
+Each option should implement the `ListKeyManagerOption` interface:
+```ts
+interface ListKeyManagerOption {
+  disabled?: boolean;
+  getLabel?(): string;
+}
+```
+
+#### Wrapping
```

Comment:
Add a short paragraph like this for typeahead?

Reply:
Can add in a follow-up PR.